packages/botonic-react/src/index.d.ts

```diff
@@ -86,4 +95,108 @@ export function msgsToBotonic(
   customMessageTypes?: CustomMessageType[]
 ): React.ReactNode
 
+export interface WebchatAppArgs {
+  theme?: ThemeProps
+  persistentMenu?: PersistentMenuProps
+  coverComponent?: CoverComponentOptions
+  blockInputs?: BlockInputOption[]
+  enableEmojiPicker?: boolean
+  enableAttachments?: boolean
+  enableUserInput?: boolean
+  enableAnimations?: boolean
+  defaultDelay?: number
+  defaultTyping?: number
+  storage?: Storage
+  onInit: onOpen
+  onClose
+  onMessage
+  appId?: string
+  visibility?: () => boolean
+}
```

Comment:
Suggestion here that applies for all the file. Since it's an `index` file, these properties should be sorted either:
- Alphabetically (all)
- By type + alphabetically (i.e: first attributes and then methods and them alphabetically)

Reply:
good one, sorted with: https://marketplace.visualstudio.com/items?itemName=henriiik.vscode-sort